src/content/en/tools/workbox/get-started/webpack.md

```diff
@@ -313,7 +313,11 @@ that they had an internet connection.
 1. Open `webpack.config.js` again.
 1. Add a `runtimeCaching` property to your Workbox configuration.
    `urlPattern` is a regular expression pattern telling Workbox which
-   URLs to store locally. `handler` defines the caching strategy that Workbox
+   URLs to store locally. *(Please note your regex will be matched
```

Comment:
Our style guide instructs us to avoid the use of "please". See [Politeness](https://developers.google.com/style/tone#politeness-and-use-of-please).

Reply:
Such a shame - the world's so harsh nowadays. C'est la vie